### PR TITLE
Regression knn model kendra

### PR DESCRIPTION
**Pull request by KendraMBach**
I explored KNN classification and discarded the model to implement Logistic Regression instead. So this branch contains the training, packaging and execution of the Logistic Regression model on the test data provided to us (test data was also normalized and standardized with steps 1 - 4). 
Check it out and let me know what you think before it's merged. 

Side note: The README.md has been updated to explain in detail the findings and reasonings behind this.

### CODE REVIEW

File: 5_regression_model/regression-knn-model.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    print(k_best)
+    '''Second Run with New K'''
+    knn = KNeighborsClassifier(n_neighbors=k_best[0])
+    knn.fit(x_train,y_train)
+
+    print("Accuracy between x_test and y_test:")
+    print(knn.score(x_test,y_test))
+
+    #print(y_test)
+    '''Confusion Matrix'''
```
Comment (CooperStansbury) [inline 5_regression_model/regression-knn-model.py]: Good call here.

**Comment by KendraMBach** [comment]
@CooperStansbury - I completely agree. I had to comment them out to ensure the training executed properly from the command line on the laptop I'm using. But I'll work on migrating the stats now for easy accessibility.
Side note, you can run the jupyter notebook now and see the visualized stats there.

**Comment by CooperStansbury** [comment]
@KendraMBach want me to merge? We can deal with this in another PR?

**Comment by KendraMBach** [comment]
@CooperStansbury - Yeah let's get this merged asap. Probably before the consolidation branch is merged, to minimize merge conflicts.

**Comment by CooperStansbury** [comment]
@KendraMBach moved content to #4. Closing this.

**Pull request closed without merging**